### PR TITLE
[ManifestLoader] Add suffix .swift to temp file created for manifest …

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
…loader

- radar: 28761873

This was causing failures sometimes in tests because the random suffix
generated could end in extensions known to swiftc/llvm like .ll, .bc etc
It would try to parse the manifest file as those files instead of a swift file
and fail with error (expected top-level entity).

### CODE REVIEW

**Comment by aciidb0mb3r** [comment]
@swift-ci please test

File: Package.swift [hunk 1]
```diff
@@ -143,7 +143,11 @@ let package = Package(
         Target(
             name: "XcodeprojTests",
             dependencies: ["Xcodeproj", "TestSupport"]),
-    ])
+    ],
+    exclude: [
+        "Tests/PackageLoadingTests/Inputs",
```
Comment (aciidb0mb3r) [inline Package.swift]: @ddunbar Do you prefer these file in fixture instead? This patch mandates that manifests end in .swift extension which are considered as sources to convention system so we either add the inputs dir in exclude or move them to fixtures.

File: Sources/PackageLoading/ManifestLoader.swift [hunk 2]
```diff
@@ -105,7 +105,7 @@ public final class ManifestLoader: ManifestLoaderProtocol {
             } catch FileSystemError.noEntry {
                 throw PackageModel.Package.Error.noManifest(inputPath.asString)
             }
-            let tmpFile = try TemporaryFile()
+            let tmpFile = try TemporaryFile(suffix: ".swift")
```
Comment (ddunbar) [inline Sources/PackageLoading/ManifestLoader.swift]: Nice find!

File: Package.swift (hunk 1 above)
Comment (ddunbar) [inline Package.swift]: I'm fine with an exclude here, it seems appropriate anyway.

**Pull request merged**